src/AsmResolver.PE/DotNet/VTableFixups/Builder/VTableFixupsDirectoryBuffer.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AsmResolver.IO;
+
+namespace AsmResolver.PE.DotNet.VTableFixups.Builder
+{
+    /// <summary>
+    /// Provides a mechanism for building the VTable fixups directory in a PE file.
+    /// </summary>
+    public class VTableFixupsDirectoryBuffer : ISegment
+    {
+        private readonly SegmentBuilder _vtableDirectory = new();
```

Comment:
Unnecessary `SegmentBuilder`, since only a single vtable fixups directory is added. Replace with the actual vtable directory?